cmd/nodecmd/dynamic_ips.go

```diff
@@ -71,6 +72,10 @@ func getPublicIPForNodesWoEIP(nodesWoEIP []models.NodeConfig) (map[string]string
 		} else {
 			publicIP, err = ec2Svc.GetInstancePublicIPs([]string{node.NodeID})
 			if err != nil {
+				if strings.Contains(err.Error(), "RequestExpired: Request has expired") {
```

Comment:
we need a separate func for this as this is a 3rd copy of the same code now

Reply:
maybe also include `err!=nil` there and just pass err into the func

Reply:
the code is slightly different depending on the caller. sometimes is returns nil, sometimes it returns err, sometimes it prints an empty line, probably we want to keep the three codes

Reply:
adding a common function to just check the condition